### PR TITLE
[DOCS-4885] Updated RUM Nav - Dashboards, Explorer, Generate Metrics

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
- Updates screenshots displaying the new RUM navigation in the Dashboard, Explorer, and Generate Metrics pages, along with their child pages
- Updates some of the text to match the updated UI accordingly
- Consolidates new and updated RUM dashboards into categories (previously, each dashboard had its own page) and updates the left nav to reflect the new categories
- Adds redirects for old dashboard links

### Motivation
[DOCS-4885](https://datadoghq.atlassian.net/browse/DOCS-4885)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-4885]: https://datadoghq.atlassian.net/browse/DOCS-4885?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/real_user_monitoring/explorer/group.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ further_reading:
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/explorer/group.md:11]: For this page, could we also update the screenshots where "Application ID" is in the query? That facet shouldn't be used as much now that we have the app selector

File: content/en/real_user_monitoring/explorer/visualize.md [hunk 2]
```diff
@@ -64,31 +64,31 @@ Visualize the top values from up to three [facet][5] according to your chosen [m
 
  **Note**: A table visualization used for one single measure and one single dimension is the same as a [top list](#top-list), just with a different display.
 
- The following RUM Analytics table shows the **top 5 URL path** for **two countries**: US and Japan consulted according to their amount of **Unique Session ID**, along with the 90th percentile of **Duration**, and over the last day:
+ The following RUM Analytics table shows the **top 5 URL paths** for **two countries**: US and Japan consulted according to their amount of **Unique Session ID**, along with the 90th percentile of **Duration**, and over the last day:
 
-{{< img src="real_user_monitoring/explorer/visualize/nested_table.png" alt="Nested table in the RUM Explorer" style="width:90%;">}}
+{{< img src="real_user_monitoring/explorer/visualize/nested_table-2.png" alt="Nested table in the RUM Explorer" style="width:90%;">}}
```
[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/real_user_monitoring/explorer/visualize.md:69]: Should work now!

File: content/en/real_user_monitoring/explorer/search.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ While RUM automatically captures events, you can also capture your own events. A
 
 To search through your RUM events, select an event type from the dropdown menu to the left of the search bar. 
 
-{{< img src="real_user_monitoring/explorer/search/rum_explorer_search.png" alt="RUM Explorer" style="width:100%;">}}
+{{< img src="real_user_monitoring/explorer/search/rum_explorer_search-2.png" alt="RUM Explorer" style="width:100%;">}}
 
 ## Search query
```
[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/real_user_monitoring/explorer/search.md:35]: updated text to the following:

> To filter on sessions produced by a specific application by real users over the past day, use the application selector from the top navigation, then create a custom query such as `@session.type:user` and set the time range to `1d`.

File: content/en/real_user_monitoring/explorer/group.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/explorer/group.md:11]: I had missed these. Thanks. Fixed!

**Comment by buraizu** [comment]
Created DOCS-5491 to review

**Review by nhientrandatadog — APPROVED** [review]

**Review by estherk15 — COMMENTED** [review]
The new dashboards look great! Left a few non-blocking suggestions
- Removing the ##Overview header
- Making the image width consistent in pages
- One spell check for facet "Viriginia"

File: content/en/real_user_monitoring/dashboards/testing_and_deployment.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  Review crash count by version, crash rate by version, and explore ongoing crashes.
+- **Errors**:
+  Review error count by version, error rate by version, and explore ongoing errors.
+- **Mobile vitals by version**:
+  For all versions, four mobile performance metrics are highlighted: slow renders, frozen frames, application start time, and memory usage.
+
+{{< img src="real_user_monitoring/dashboards/dashboard-deployment-mobile.png" alt="Out-of-the-box mobile deployment dashboard" style="width:100%" >}}
+
+For more information about the data displayed, see our documentation for each platform: [iOS RUM][5], [Android RUM][6], [React Native RUM][7], and [Flutter RUM][8].
+
```
[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/real_user_monitoring/dashboards/testing_and_deployment.md:72]: Removed - thanks for the reminder!

**Comment by rtrieu** [comment]
@estherk15 Thanks so much for the quick review and suggestions - accepted them all!

File: content/en/real_user_monitoring/dashboards/testing_and_deployment.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+[1]: https://app.datadoghq.com/dash/integration/30697/synthetics---browser-test-performance
+[2]: /real_user_monitoring/browser/data_collected/
+[3]: /synthetics/browser_tests/
+[4]: /real_user_monitoring/browser/#initialization-parameters
+[5]: /real_user_monitoring/ios/data_collected/
+[6]: /real_user_monitoring/android/data_collected/
+[7]: /real_user_monitoring/reactnative/data_collected/
+[8]: /real_user_monitoring/flutter/data_collected/
+
+<!-- [4]: /synthetics/cicd_integrations/
```
Comment (estherk15) [inline content/en/real_user_monitoring/dashboards/testing_and_deployment.md:86]: Sorry should have highlighted this before if this needs to be cleaned up or not.

**Review by estherk15 — APPROVED** [review]
Found one more possible cleanup, but it's non-blocking.

**Review by xgouchet — APPROVED** [review]

**Review by xgouchet — APPROVED** [review]

File: content/en/real_user_monitoring/_index.md [hunk 6]
```diff
@@ -128,13 +128,13 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
 
 ## Explore Datadog RUM
 
-Access RUM by navigating to [**UX Monitoring > RUM Applications**][1].
+Access RUM by navigating to [**UX Monitoring > Real User Monitoring**][1].
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/_index.md:131]: can we change the hyperlink to go to https://app.datadoghq.com/rum/performance-monitoring ?

File: content/en/real_user_monitoring/dashboards/_index.md [hunk 7]
```diff
@@ -10,22 +10,19 @@ further_reading:
 
 ## Overview
 
-When you create a RUM application, Datadog [collects data][1] and generates dashboards about your application's performance, errors, resources, user sessions, and mobile performance. 
+When you create a RUM application, Datadog [collects data][1] and generates dashboards about your application's performance, errors, resources, and user sessions. 
 
-{{< img src="real_user_monitoring/dashboards/rum_application_overview_dashboard.png" alt="RUM Application Overview page" style="width:90%;" >}}
+{{< img src="real_user_monitoring/dashboards/rum_application_overview_dashboard-3.png" alt="RUM Application Overview page" style="width:90%;" >}}
 
 Access your RUM dashboards by filtering for `RUM` in the search query of the [**Dashboard List**][2] or by clicking on a dashboard's **Application Overview** in the [**RUM Applications** page][3].
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/dashboards/_index.md:17]: can we rephrase the second part of these instructions? something like:
... or from your application summary pages (**Real User Monitoring > Performance Summary** and **Real User Monitoring > Analytics Summary**)

not sure what our standard link syntax is, but that's the gist

File: content/en/real_user_monitoring/explorer/_index.md [hunk 8]
```diff
@@ -19,15 +19,21 @@ further_reading:
 
 The Real User Monitoring (RUM) Explorer allows you to examine data collected from your applications and granular information about your RUM events. 
 
-Click the **Sessions** tab under **UX Monitoring** > **RUM Applications** to access the [RUM Explorer][1] and investigate any questions you have about your applications' health and usage. 
+From the menu, click **Sessions & Replays** under **UX Monitoring** > **Real User Monitoring** to access the [RUM Explorer][1] and investigate any questions you have about your applications' health and usage.
```
Comment (nhientrandatadog) [inline content/en/real_user_monitoring/explorer/_index.md:22]: can we update the URL to this? https://app.datadoghq.com/rum/sessions

Comment (nhientrandatadog) [reply content/en/real_user_monitoring/explorer/_index.md:22]: the old URL does redirect, but it would be nice to link to the updated one

**Review by nhientrandatadog — APPROVED** [review]
A few comments but otherwise looks good!

**Pull request merged**